### PR TITLE
#162,166,167 - Add logo, links and badges to readme.md

### PR DESCRIPTION
**Pull request by rockfridrich**
#162, #166, #167

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,9 +67,7 @@ let response = try SendRawTransactionProcedure(
         network: network,
         senderKey: sender,
         recipientAddress: recipient,
-        weiAmount: EthNumber(
-            value: amount
-        )
+        weiAmount: amount
```
Comment (biboran) [inline README.md]: wat

Comment (biboran) [reply README.md]: are you sure this is right?

Comment (rockfridrich) [reply README.md]: The amount is already an `EthNumber`
```swift
let amount: BytesScalar = EthNumber(
     hex: "0xde0b6b3a7640000" // 10^18 in hex that represents 1 ETH
)
```

Comment (rockfridrich) [reply README.md]: @biboran

Comment (biboran) [reply README.md]: You are right

**Pull request merged**